docs/specs/markdown.yml

```diff
@@ -170,3 +170,30 @@ outputs:
   docs/a.json: |
     { "content": "<p><a href=\"http://*.windowsupdate.microsoft.com\">http://*.windowsupdate.microsoft.com</a></p>" }
   build.manifest:
+---
+# validate bookmarks in markdown documents
+inputs:
+  docfx.yml:
+  docs/a.md: |
+    # title 1
+    [link to title 1](#title-1)
+    [link to title 2](#title-2)
+    [link to title 3](#title-3)
+    [link to html title](#html-title)
+    [link to yaml](c.yml#no-such-bookmark) does not warn
+    ## title 2
+    <h2 id="html-title">title</h2>
+  docs/b.md: |
+    [link to title 2](a.md#title-2)
+    [link to title 3](a.md#title-3)
+  docs/c.yml: |
+    #YamlMime:LandingData
+    title: test
+outputs:
+  docs/a.json:
+  docs/b.json:
+  docs/c.json:
+  build.log: |
+    ["warning","bookmark-not-found","Cannot find bookmark '#title-3' in 'docs/a.md'","docs/b.md"]
```

Comment:
yes, this is just a warning

---
In reply to: [205678643](https://github.com/dotnet/docfx/pull/3121#discussion_r205678643) [](ancestors = 205678643)

Reply:
It will be a simple fix in the source

---
In reply to: [205694225](https://github.com/dotnet/docfx/pull/3121#discussion_r205694225) [](ancestors = 205694225,205678643)